regression-test/suites/inverted_index_p0/test_ip_cidr_search_with_inverted_index.groovy

```diff
@@ -40,95 +63,107 @@ suite("test_ip_cidr_search_with_inverted_index", "nonConcurrent"){
-                order_qt_sql """ ${sqlStr} """
-            } finally {
-                GetDebugPoint().disableDebugPointForAllBEs(checkpoints_name)
-            }
-            ++i
+        // with inverted index query
+        // If we use common expr pass to inverted index , we should set enable_common_expr_pushdown = true
+        sql """ set enable_common_expr_pushdown = true; """
+        sql """ set enable_inverted_index_query=true; """
+        sql """ set inverted_index_skip_threshold = 0; """ // set skip threshold to 0
+
+        // select ipv6 in ipv4 cidr
+        qt_sql_with_ii_0 """  select id, ipv4, ipv6, is_ip_address_in_range(ipv6, '255.255.255.255/12') from tc_ip_cidr_search_with_inverted_index where is_ip_address_in_range(ipv6, '255.255.255.255/12') order by id; """
+        // select ipv6 in ipv6 cidr
+        qt_sql_with_ii_1 """  select id, ipv4, ipv6, is_ip_address_in_range(ipv6, 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff/12') from tc_ip_cidr_search_with_inverted_index where is_ip_address_in_range(ipv6, 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff/12') order by id; """
+        qt_sql_with_ii_2 """  select id, ipv4, ipv6, is_ip_address_in_range(ipv6, '::ffff:192.168.0.4/128') from tc_ip_cidr_search_with_inverted_index where is_ip_address_in_range(ipv6, '::ffff:192.168.0.4/128') order by id; """
+        qt_sql_with_ii_3 """  select id, ipv4, ipv6, is_ip_address_in_range(ipv6, '2001:16a0:2:200a::2/64') from tc_ip_cidr_search_with_inverted_index where is_ip_address_in_range(ipv6, '2001:16a0:2:200a::2/64') order by id; """
+
+        // select ipv4 in ipv6 cidr
+        qt_sql_with_ii_4 """  select id, ipv4, ipv6, is_ip_address_in_range(ipv4, 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff/12') from tc_ip_cidr_search_with_inverted_index where is_ip_address_in_range(ipv4, 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff/12') order by id; """
+        // select ipv4 in ipv4 cidr
+        qt_sql_with_ii_5 """  select id, ipv4, ipv6, is_ip_address_in_range(ipv6, '255.255.255.255/12') from tc_ip_cidr_search_with_inverted_index where is_ip_address_in_range(ipv4, '255.255.255.255/12') order by id; """
+        qt_sql_with_ii_6 """  select id, ipv4, ipv6, is_ip_address_in_range(ipv4, '127.0.0.0/8') from tc_ip_cidr_search_with_inverted_index where is_ip_address_in_range(ipv4, '127.0.0.0/8') order by id; """
+        qt_sql_with_ii_7 """  select id, ipv4, ipv6, is_ip_address_in_range(ipv4, '192.168.100.0/24') from tc_ip_cidr_search_with_inverted_index where is_ip_address_in_range(ipv4, '192.168.100.0/24') order by id; """
+
+        // select in null cidr
+        qt_sql_with_ii_8 """  select id, ipv4, ipv6, is_ip_address_in_range(ipv4, null) from tc_ip_cidr_search_with_inverted_index where is_ip_address_in_range(ipv4, null) order by id; """
+        qt_sql_with_ii_9 """  select id, ipv4, ipv6, is_ip_address_in_range(ipv6, null) from tc_ip_cidr_search_with_inverted_index where is_ip_address_in_range(ipv6, null) order by id; """
+
+
```

Comment:
string uuid= genUUID();
for () {
   sql "
     select uuid, id, ipv4, ipv6 from tc_ip_cidr_search_with_inverted_index where is_ip_address_in_range(ipv6, '255.255.255.255/12') order by id;
    "
} catch () {
    getAllProfile
    profileID =    filterQueryByUUID(uuid)
    getProfile(    profileID)
}